IPython/core/display.py

```diff
@@ -512,31 +512,54 @@ def _repr_svg_(self):
 
 
 class JSON(DisplayObject):
-    """JSON expects a JSON-able dict or list
-    
-    not an already-serialized JSON string.
-    
-    Scalar types (None, number, string) are not allowed, only dict or list containers.
-    """
+
```

Comment:
@minrk Likely an oversight when __init__ was added and the docstring contents were moved to its docstring. I agree it would be good to have the class docstring as well.

:tada: for writing docstrings @aleury